language/move-stdlib/modules/Option.move

```diff
@@ -110,7 +110,10 @@ module Option {
 
     /// Return the value inside `t` if it holds one
     /// Return `default` if `t` does not hold a value
-    public fun get_with_default<Element: copy + drop + store>(t: &Option<Element>, default: Element): Element {
+    public fun get_with_default<Element: copy + drop>(
```

Comment:
Do we need `copy` here?

Reply:
Yes because it dereferences out of the Option (it isn't unwrap_with_default)